recipes-devtools/maccalc/maccalc.bb

```
@@ -0,0 +1,21 @@
+#
+# This file was derived from the 'Hello World!' example recipe in the
+# Yocto Project Development Manual.
+#
+SUMMARY = "MAC address manipulation utility"
+
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://main.c;md5=641b6faf83ff4caf4264dde86a5ef503"
```

I just noticed some more room for nitpicking:
What I've seen in Yocto is that the license checksum is calculated over a few lines of a file, to just catch possible changes of the license (not in the other sections of text) when changing the sources:
```suggestion
LIC_FILES_CHKSUM = "file://main.c;beginline=6;endline=8;md5=badf00d"
```
See e.g. https://git.openembedded.org/meta-openembedded/tree/meta-oe/recipes-devtools/dmalloc/dmalloc_5.5.2.bb?h=master#n16